docs/api.md

```diff
@@ -31,7 +47,7 @@ consistency model that Vitess expects of a lockserver.</p>
 <tbody>
 <tr>
 <td>
-<code>apiVersion</code><br/>
+<code>apiVersion</code></br>
```

Comment:
This looks wrong. There is a closing `</br>` tag. If a line break is the intention it should probably be the line break `<br/>`.